src/global-messages/GlobalMessage.tsx

```diff
@@ -60,39 +60,37 @@ const GlobalMessage = ({
         .filter((gm) => isWithinTimeRange(gm, now))
         .map((globalMessage: GlobalMessageType) => {
           const message = getTextForLanguage(globalMessage.body, language);
+          const onPressAction = globalMessage.link
+            ? {text: globalMessage.link, url: globalMessage.link}
+            : undefined;
           if (!message) return null;
-          return (
-            <>
-              {globalMessage.subtle ? (
-                <MessageInfoText
-                  key={globalMessage.id}
-                  type={globalMessage.type}
-                  message={message}
-                  textColor={textColor}
-                  isMarkdown={true}
-                  style={style}
-                  testID="globalMessage"
-                />
-              ) : (
-                <MessageInfoBox
-                  key={globalMessage.id}
-                  style={style}
-                  title={getTextForLanguage(
-                    globalMessage.title ?? [],
-                    language,
-                  )}
-                  message={message}
-                  type={globalMessage.type}
-                  isMarkdown={true}
-                  onDismiss={
-                    globalMessage.isDismissable && !includeDismissed
-                      ? () => dismissGlobalMessage(globalMessage)
-                      : undefined
-                  }
-                  testID="globalMessage"
-                />
-              )}
-            </>
+
+          return globalMessage.subtle ? (
+            <MessageInfoText
+              key={globalMessage.id}
+              type={globalMessage.type}
+              message={message}
+              textColor={textColor}
+              isMarkdown={true}
+              style={style}
+              testID="globalMessage"
+            />
+          ) : (
+            <MessageInfoBox
+              key={globalMessage.id}
+              style={style}
+              title={getTextForLanguage(globalMessage.title ?? [], language)}
+              message={message}
+              type={globalMessage.type}
+              isMarkdown={true}
+              onDismiss={
+                globalMessage.isDismissable && !includeDismissed
+                  ? () => dismissGlobalMessage(globalMessage)
+                  : undefined
+              }
+              onPressConfig={onPressAction}
```

Comment:
Remember to add to documentation that links only works for non-subtle global messages.

Reply:
Documentation updated